### ISSUE TITLE
Empty results throw error instead of returning empty dataframe

### ISSUE THREAD

**Issue by miflower** [Azure/azure-kusto-python#112]
When using pandas, the dataframe_from_result_table call throws an error if it is passed a result set that has a blank result set.

#103 introduced this problem [Lines 13 & 14](https://github.com/Azure/azure-kusto-python/blob/master/azure-kusto-data/azure/kusto/data/helpers.py#L13) throw an error if the parameter evaluates to False.  As empty result tables evaluate to false this results in any blank result set throwing a ValueError.

Working code:
```
response = client.execute("DB", "print 'a'")
return [dataframe_from_result_table(x) for x in response.primary_results]
```

Broken code:
```
response = client.execute("DB", "print 'a' | take 0")
return [dataframe_from_result_table(x) for x in response.primary_results]
```

**Comment by toshetah** [comment]
Hey @miflower,

Thanks for reporting!
We've checked in a fix, and published version 0.0.24 .
Can you please confirm that it solves the issue?

Thanks,
Tomer

**Comment by miflower** [comment]
Hey @toshetah,
That did the trick.  Thanks for the quick response.

Thanks,
Mike
